wasm/ro/chapter_3.yaml

```
@@ -0,0 +1,263 @@
+- title: Capitolul 3 - Invocare dinamică
+  content_markdown: >
+    În acest capitol vom discuta despre cum putem crea propriile funcții
+    dinamice la runtime pentru a le apela din WebAssembly.
+- title: De ce invocare dinamică?
+  content_markdown: >
+    Am văzut în primele capitole că putem foarte ușpr să scriem manual propriile
+    funcții JavaScript în fișierele noaste `.js`. Așa că
+
+    de ce am crea funcții dinamice?
+
+
+    * Majoritatea oamenilor aleg WebAssembly pentru a scăpa de JavaScript. Crearea
+    funcțiilor JavaScript dinamice ne permite să folosim
+      biblioteci în limbajul nativ, care să se ocupe de partea dificilă de a crea
+      dinamic legăturile fără să reinventeze roata.
+    
+    * Simplifică configurarea și lansarea modulelor WebAssembly la un minim absolut.
+    Avem nevoie doar de capacitatea de a înregistra funcții noi
+    și de a le apela în loc să specificăm o serie de funcții JavaScript scrise manual.
+
+    * Este dificil pentru JavaScript să știe ce dependențe are modulul său,
+    permițând programului WebAssembly să-și creeze propriile funcții care să facă
+    exact ceea ce este nevoie.
+- title: Recapitularea înregistrării
+  content_markdown: >
+    În primul capitol am scris manual funcțiile noastre de log pentru a efectua înregistrarea
+    cu `console.log`. Să vedem cum am putea face acest lucru dinamic.
+
+
+    * Avem nevoie de o modalitate de a înregistra o funcție la runtime.
+
+    * Avem nevoie de o modalitate de a apela acea funcție.
+
+    * Avem nevoie ca acea funcție dinamică să aibă acces la memorie (de exemplu,
+    pentru a extrage octeți utf-8).
+- title: Înregistrarea funcțiilor
+  content_markdown: >
+    Pentru a înregistra o funcție avem nevoie de două lucruri:
+
+
+    * trebuie să pasăm corpul funcției pe care dorim să o creăm în JavaScript
```

```suggestion
    * trebuie să transmitem corpul funcției pe care dorim să o creăm în JavaScript
```
Consider că "transmitem" este un echivalent mai bun în limba română.